bgpd/bgp_vty.c

```diff
@@ -10844,6 +10844,12 @@ static int bgp_show_summary(struct vty *vty, struct bgp *bgp, int afi, int safi,
 						    peer->established);
 				json_object_int_add(json_peer, "connectionsDropped",
 						    peer->dropped);
+				if (peer->desc)
+					json_object_string_add(
+						json_peer, "desc", peer->desc);
+				else
+					json_object_string_add(json_peer,
+							       "desc", "N/A");
```

Comment:
This should be an empty string, not "N/A"

Reply:
@qlyoung, please discuss with @ton31337 about that.
I think you re right.

Reply:
For JSON, I agree with @qlyoung. For non-JSON, I still think that N/A or so is needed to avoid empty output (might confuse someone).

Reply:
Correct

Reply:
update done. lets check if CI is happy, first.

Reply:
now that CI is happy, lets approve this ticket please.